tests/app.test.jsx

```diff
@@ -22,6 +22,8 @@ jest.mock('../client/src/components/Overview/Gallery', () => () => (<div>Placeho
 jest.mock('../client/src/components/Q&A/QuestionsAndAnswers', () => () => (<div>Placeholder Questions And Answers</div>));
 jest.mock('../client/src/components/Q&A/questions.css', () => () => (<div>Placeholder Questions And Answers Style</div>));
 jest.mock('../client/src/components/Related/RelatedItems', () => () => (<div>Placeholder Questions And Answers</div>));
+jest.mock('../client/src/components/Reviewlist/reviewlist.css', () => () => (<div>Review List Style</div>));
```

Comment:
@kyang9021 FYI I added this line of code to app.test so that the tests can pass with the css files